components/input/input.css

```diff
@@ -0,0 +1,145 @@
+.text-area-container {
+
+    display: flex;
+    flex-direction: column;
+    padding: 1rem;
+    background-color: var(--lightgray-color);
+    width: 25rem;
+    height: 15rem;
+    border-radius: 0.3rem;
+
+}
+
+.text-area {
+    padding: 1rem;
+    font-size: 1.2rem;
+    border: none;
+    box-shadow: 0 0 5px var(--lightgray-color);
+}
+
+.text-area:focus {
+    outline: none;
```

Comment:
Don't remove `outline` it will kill accessibility, You can use `outline: 2px transparent solid;` (change the width as according.)

Reply:
yes made that change